pkg/cmd/roachtest/cluster.go

```diff
@@ -106,7 +106,13 @@ func (v *encryptValue) Type() string {
 	return "string"
 }
 
-var errBinaryOrLibraryNotFound = errors.New("binary or library not found")
+type errBinaryOrLibraryNotFound struct {
+	binary string
+}
+
+func (e errBinaryOrLibraryNotFound) Error() string {
+	return fmt.Sprintf("binary of library %q not found (or was not executable)", e.binary)
```

Comment:
```suggestion
	return fmt.Sprintf("binary or library %q not found (or was not executable)", e.binary)
```

Reply:
Thanks! That's what I get for git pushing directly after rage fixing this.